### ISSUE TITLE
Implementing Lottery Ticket Hypothesis

### ISSUE THREAD

**Issue by ajktym94** [VainF/Torch-Pruning#95]
Is there any way I could implement the [Lottery Ticket Hypothesis](https://arxiv.org/abs/1803.03635)?
With v0.2.8, I was able to implement it. I obtained the pruning_idxs using the model being pruned and executed a pruning plan based on these indices on the pretrained model. However the accuracy went too low. The idea of `ignored_layers` in v1.0.0 looks like a good way to proceed. However, I need to find the pruning indices using the  model being trained and then execute it on the pretrained model. So essentially, this avoids the need to copy the weights. The pretrained model would be pruned and then it can be used as the model being trained. Does anyone know how this can be done?
